bridge/github/export.go

```diff
@@ -111,26 +111,33 @@ func (ge *githubExporter) ExportAll(repo *cache.RepoCache, since time.Time) (<-c
 		allBugsIds := repo.AllBugsIds()
 
 		for _, id := range allBugsIds {
-			b, err := repo.ResolveBug(id)
-			if err != nil {
-				out <- core.NewExportError(err, id)
+			select {
+			case <-ctx.Done():
+				out <- core.NewExportError(ctx.Err(), "")
```

Comment:
Do as you prefer, but maybe you need a new result type for an interrupted process ?

Reply:
Yes that would be nice !